jmclient/jmclient/fidelity_bond.py

```diff
@@ -3,7 +3,7 @@
 import json
 from jmbitcoin import ecdsa_sign, ecdsa_verify
 from jmdaemon import fidelity_bond_sanity_check
-
+import binascii
```

Comment:
Worth noting here, we have replaced all imports of `binascii` in the codebase with `bintohex` and `hextobin` from `jmbase` - but iiuc  you want this because you want to convert binary to hex-encoded binary, not a string.